### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/prodeveloper/social-share/pull/65/commits/72b8d134140df4ba653401d8af39bb141ab49e65?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/social-share.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "chencha/share": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ jobs:
         strategy:
             fail-fast: true
             matrix:
-                laravel: [^6, ^7, ^8]
-                php: [7.4, 8.0]
+                laravel: [^6, ^7, ^8, 9.0]
+                php: [7.4, 8.0, 8.0]
```
Comment (jasonmccreary) [inline .github/workflows/build.yml]: Seems the PHP version got duplicated. We'll clean that up for next time.

Comment (datashaman) [reply .github/workflows/build.yml]: Thanks!

**Comment by datashaman** [comment]
@jasonmccreary a few things i'm changing, which you might wanna tweak in Shift:

- Changing the version spec in composer.json from ^9.0 to ^9 to match the others (not a biggie, both work, I imagine).
- Adding the matching orchestra/testbench version in dev-requirements as ^7 (their versions are off by 2). _This change is essential imo_.
- Adding explicitly separate ^8.1 in composer.json's require.php spec.
- Fixing the duplicated PHP version in build file and changing ^9.0 to ^9 like the other versions.

The commit might be clearer. :)

**Pull request merged**